qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -66,7 +66,7 @@ class BaseSpecification extends Specification {
 
     public static strictIntegrationTesting = false
 
-    Map<String, List<String>> resourceRecord = [:]
+    private static Map<String, List<String>> resourceRecord = [:]
```

Comment:
I don't understand how this solves the problem :(

Reply:
> I don't understand how this solves the problem :(

The resource leak check is changed to cover the entire gradle task run instead of per test spec. With (potentially) parallel test specs this check is no longer viable at that level.

Reply:
I updated the function names to better reflect how they are called with this change.